include/benchmark/benchmark.h

```diff
@@ -413,9 +411,11 @@ enum TimeUnit { kNanosecond, kMicrosecond, kMillisecond };
 // calculated automatically to the best fit.
 enum BigO { oNone, o1, oN, oNSquared, oNCubed, oLogN, oNLogN, oAuto, oLambda };
 
+typedef uint64_t IterationT;
```

Comment:
nit: Should be `Iterations` or `IterationCount`.

Reply:
Let's go with `IterationCount`